On linux it tries to download some archive which is forbidden in the sandbox:

``` 
-- Using src='https://github.com/manticoresoftware/re2/archive/2015-06-01.zip'
CMake Error at re2-subbuild/re2-populate-prefix/src/re2-populate-stamp/download-re2-populate.cmake:170 (message):
  Each download failed!
    error: downloading 'https://github.com/manticoresoftware/re2/archive/2015-06-01.zip' failed
          status_code: 6
          status_string: "Couldn't resolve host name"
          log:
          --- LOG BEGIN ---
          Could not resolve host: github.com
  Closing connection 0
```

--- a/maintainers/maintainer-list.nix
+++ b/maintainers/maintainer-list.nix
@@ -6221,6 +6221,12 @@
     githubId = 117874;
     name = "Jeroen de Haas";
   };
+  jdelStrother = {
+    email = "me@delstrother.com";
+    github = "jdelStrother";
+    githubId = 2377;
+    name = "Jonathan del Strother";
+  };
   jdreaver = {
     email = "johndreaver@gmail.com";
     github = "jdreaver";

--- a/pkgs/servers/search/manticoresearch/default.nix
+++ b/pkgs/servers/search/manticoresearch/default.nix
@@ -1,0 +1,70 @@
+{ lib, stdenv, fetchFromGitHub, fetchurl
+, bison, cmake, flex, pkg-config
+, boost, icu, libstemmer, mariadb-connector-c, re2
+}:
+let
+  columnar = stdenv.mkDerivation rec {
+    pname = "columnar";
+    version = "c16-s5"; # see NEED_COLUMNAR_API/NEED_SECONDARY_API in Manticore's GetColumnar.cmake
+    src = fetchFromGitHub {
+      owner = "manticoresoftware";
+      repo = "columnar";
+      rev = version;
+      sha256 = "sha256-iHB82FeA0rq9eRuDzY+AT/MiaRIGETsnkNPCqKRXgq8=";
+    };
+    nativeBuildInputs = [ cmake ];
+    cmakeFlags = [ "-DAPI_ONLY=ON" ];
+    meta = {
+      description = "A column-oriented storage and secondary indexing library";
+      homepage = "https://github.com/manticoresoftware/columnar/";
+      license = lib.licenses.asl20;
+      platforms = lib.platforms.all;
+    };
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "manticoresearch";
+  version = "5.0.3";
+
+  src = fetchFromGitHub {
+    owner = "manticoresoftware";
+    repo = "manticoresearch";
+    rev = version;
+    sha256 = "sha256-samZYwDYgI9jQ7jcoMlpxulSFwmqyt5bkxG+WZ9eXuk=";
+  };
+
+  nativeBuildInputs = [
+    bison
+    cmake
+    flex
+    pkg-config
+  ];
+
+  buildInputs = [
+    boost
+    columnar
+    icu.dev
+    libstemmer
+    mariadb-connector-c
+    re2
+  ];
+
+  postPatch = ''
+    sed -i 's/set ( Boost_USE_STATIC_LIBS ON )/set ( Boost_USE_STATIC_LIBS OFF )/' src/CMakeLists.txt
+
+    # supply our own packages rather than letting manticore download dependencies during build
+    sed -i 's/^with_get/with_menu/' CMakeLists.txt
+  '';
+
+  cmakeFlags = [
+    "-DWITH_GALERA=0"
+  ];
+
+  meta = with lib; {
+    description = "Easy to use open source fast database for search";
+    homepage = "https://manticoresearch.com";
+    license = licenses.gpl2;
+    maintainers = with maintainers; [ jdelStrother ];
+    platforms = platforms.all;
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -20914,6 +20914,7 @@ with pkgs;
     boost = boost175;
   };
 
+  manticoresearch = callPackage ../servers/search/manticoresearch { };
 
   marisa = callPackage ../development/libraries/marisa {};
 

